src/components/EntityInfo/EntityHeader.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <header id="entity-header">
+    <template v-if="!!businessId">
+      <!-- Title -->
+      <div id="entity-legal-name" aria-label="Entity Legal Name">
+        {{ getEntityName || 'Unknown Name' }}
+      </div>
+
+      <!-- Subtitle -->
+      <div>
+        <span id="business-description">{{ businessDescription }}</span>
+
+        <span id="limited-restoration" class="ml-3" v-if="isInLimitedRestoration">
+          <span class="ml-3">Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}</span>
+          <v-chip class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
+            small label text-color="white">LIMITED RESTORATION</v-chip>
+        </span>
+
+        <span id="authorized-to-continue-out" v-if="isAuthorizedToContinueOut">
+          <v-chip class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
+            small label text-color="white">AUTHORIZED TO CONTINUE OUT</v-chip>
+        </span>
+      </div>
+    </template>
+
+    <template v-if="!!tempRegNumber">
+      <!-- Title -->
+      <div id="ia-reg-name" aria-label="Incorporation Application or Registration Entity Name">
+        {{ getEntityName || 'Unknown Name' }}
+      </div>
+
+      <!-- Subtitle -->
+      <div id="ia-reg-description" aria-label="Incorporation Application or Registration Description">
+        {{ iaRegDescription }}
+      </div>
+    </template>
```

Comment:
Here's the template for the EntityHeaderTempReg

Reply:
If you don't want to see this template when you're in this file, code-fold it.

At the moment, I prefer to keep like content together instead of managing more component files. Also see my previous comment on this file.